test/OpenTelemetry.Instrumentation.Runtime.Tests/RuntimeMetricsTests.cs

```diff
@@ -37,6 +38,90 @@ public void RuntimeMetricsAreCaptured()
             meterProvider.ForceFlush(MaxTimeToAllowForFlush);
             Assert.True(exportedItems.Count > 1);
             Assert.StartsWith(MetricPrefix, exportedItems[0].Name);
+
+            var gcCountMetric = exportedItems.First(i => i.Name == "process.runtime.dotnet.gc.collections.count");
```

Comment:
I'd suggest to split the tests rather than having an single test covering entire instrumentations. That way, if things fail, its easy to tell from the name of the test which part is failing.

Reply:
I tried to split up the tests. However, in that case, a garbage collection would have occurred before any of the unit tests. I will not be able to test the case that "before garbage collection happens, certain metrics won't exists". Some context: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/510#discussion_r923963846

Reply:
I think I should skip testing that scenario, because it depends on Runtime very much. Even the current unit tests could fail if GC decides to run once before the unit tests.

Reply:
There is no need to test for that scenario, as it is hard to simulate that and there is no value testing it.